src/css/panel.base.scss

```diff
@@ -31,3 +31,12 @@ panel-plugin-export-manager-panel {
     }
   }
 }
+
+.disabled-button {
+  opacity: 50%;
```

Comment:
I hope we don't use custom design approaches. There are plenty styles from grafana itself